xdsl_torch/utils/import_program.py

```diff
@@ -1,13 +1,64 @@
+from itertools import zip_longest
+from typing import Any
+
 import torch
 from xdsl.builder import ImplicitBuilder
-from xdsl.dialects import func
-from xdsl.dialects.builtin import TensorType
+from xdsl.dialects import arith, func
+from xdsl.dialects.builtin import BoolAttr, FloatAttr, IntegerAttr, TensorType
 from xdsl.ir import SSAValue
 
 from xdsl_torch.dialects.torch_mapping import XDSL_TORCH_OPS
 from xdsl_torch.utils.type_mapping import TORCH_DTYPE_TO_XDSL_TYPE
 
 
+def literal_to_ssa(value: Any) -> tuple[str, SSAValue]:
+    match value:
+        case int():
+            attr = IntegerAttr.from_int_and_width(value, 32)
+        case float():
+            attr = FloatAttr(value, 32)
+        case bool():
+            attr = BoolAttr.from_bool(value)
+        case _:
+            raise NotImplementedError(
+                f"Default values not implemented for type {type(value)}"
+            )
+
+    new_const = arith.ConstantOp(attr, attr.get_type())
+    new_name = f"{type(value).__name__}{value}"
+    new_const.result.name_hint = new_name
+
+    return new_name, new_const.result
+
+
+def get_op_operands(
+    node: torch.fx.Node, xdsl_nodes: dict[str, SSAValue]
+) -> list[SSAValue]:
+    arguments: list[tuple[torch.Argument, Any]] = list(
+        zip_longest(node.target._schema.arguments, node.args)  # type: ignore
+    )
+    operands: list[SSAValue] = []
+
+    for arg_spec, arg_value in arguments:
+        if type(arg_value) is torch.fx.Node:
+            if arg_value.name not in xdsl_nodes:
+                raise Exception(
```

Comment:
Plain Exceptions are usually too broad. Can you construct a torch object that would cause this? If yes, this should probably be a ValueError (or a TorchConversionError, subclass of ValueError or DiagnosticException), with an accompanying pytest. If no, this should probably be an assert instead. Similarly below.